aiokafka/coordinator/assignors/sticky/sticky_assignor.py

```diff
@@ -70,43 +94,49 @@ class StickyAssignorUserDataV1(Struct):
 
 
 class StickyAssignmentExecutor:
-    def __init__(self, cluster, members):
+    def __init__(
+        self,
+        cluster: ClusterMetadata,
+        members: Dict[str, StickyAssignorMemberMetadataV1],
+    ) -> None:
         self.members = members
         # a mapping between consumers and their assigned partitions that is updated
         # during assignment procedure
-        self.current_assignment = defaultdict(list)
+        self.current_assignment: Dict[str, List[TopicPartition]] = defaultdict(list)
         # an assignment from a previous generation
-        self.previous_assignment = {}
+        self.previous_assignment: Dict[TopicPartition, ConsumerGenerationPair] = {}
         # a mapping between partitions and their assigned consumers
-        self.current_partition_consumer = {}
+        self.current_partition_consumer: Dict[TopicPartition, str] = {}
         # a flag indicating that there were no previous assignments performed ever
         self.is_fresh_assignment = False
         # a mapping of all topic partitions to all consumers that can be assigned to
         # them
-        self.partition_to_all_potential_consumers = {}
+        self.partition_to_all_potential_consumers: Dict[TopicPartition, List[str]] = {}
         # a mapping of all consumers to all potential topic partitions that can be
         # assigned to them
-        self.consumer_to_all_potential_partitions = {}
+        self.consumer_to_all_potential_partitions: Dict[str, List[TopicPartition]] = {}
         # an ascending sorted set of consumers based on how many topic partitions are
         # already assigned to them
-        self.sorted_current_subscriptions = SortedSet()
+        self.sorted_current_subscriptions: SortedSet[
+            Tuple[str, Tuple[TopicPartition, ...]]
+        ] = SortedSet()
```

Comment:
_nit:_ Here and in other places, it would be nice to avoid breaking in inner expression when it's possible to break in outer, the current version is difficult to read: we have a statement `lvalue: type = expression` and the break is inside of "word" `type` instead of being between word. I saw this or similar problem was solved in some recent version of ruff. Maybe its update will be enough?

Reply:
Something like this was solved in ruff 3 or 4, but not for this case. May be use `# fmt: skip`?

Reply:
Not very important, at your discretion